Uppsala/UppEt3.xml

```diff
@@ -0,0 +1,557 @@
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="la">Titulus usque ad 3iam lineam Johannis episcopi 
+                                    Constantinopolis metropoleos</q>
+                                </item>
+                                
+                                <item xml:id="e5">
+                                    <locus target="ivv"/>
+                                    <desc>Illegible stamp. It is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                </item>
+                                
+                                <item xml:id="e6">
+                                    <locus target="1r"/>
+                                    <desc>French note in the upper right corner. It is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="la">L'apocalipse</q>
+                                </item>
+                                
+                                <item xml:id="e7">
+                                    <locus target="157r"/>
+                                    <desc>French note in the bottom margin. It is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="la">Du <sic corresp="JK">kalendrier</sic> <sic corresp="JK">abissin</sic></q>
+                                </item>
+                                
+                                <item xml:id="e8">
+                                    <desc>Latin chapter numbers written in a European hand throughout <ref target="#ms_i1"/>. 
+                                        They are not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                </item>
+
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">
+                                    As noted by <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">31</citedRange></bibl>,
```

Comment:
Since good images are available, it is better to describe the binding according to Bm scheme (which would make the part  more clear and informative). To refer to the leather cover and the fact that it is tooled, you do not need to quote the catalogues.

Reply:
I don't have any experience of using the Bm scheme for binding descriptions, so if you have time, I would appreciate your feedback, especially if there is anything essential that I missed. Although we have good images, I still don't understand exactly how the leather patches on the spine connect to the rest.

Reply:
Ok, we can do this.